libs/utils/WordPressUtils/src/main/java/org/wordpress/android/util/helpers/WPHtmlTagHandler.java

```diff
@@ -33,27 +34,30 @@ public void handleTag(final boolean opening, final String tag, Editable output,
     }
 
     private void handleListTag(Editable output) {
-        if (mListParents.lastElement().equals("ul")) {
-            output.append("\n");
-            String[] split = output.toString().split("\n");
-            int start = 0;
-            if (split.length != 1) {
-                int lastIndex = split.length - 1;
-                start = output.length() - split[lastIndex].length() - 1;
+        int size = mListParents.size();
+        if (size > 0) {
```

Comment:
good call in checking for size > 0, and having it assigned to a var so it's not evaluated every time. Thank you!